share/wake/lib/system/job.wake

```diff
@@ -86,23 +89,40 @@ export data Persistence =
 
 # A Plan describes a not-yet-executed Job
 tuple Plan =
-  export Label:        String       # The label used when showing the command during execution
-  export Command:      List String  # The command-line arguments (the first is the command to run)
-  export Visible:      List Path    # Only these files should be available to the command
-  export Environment:  List String  # KEY=VALUE environment variables fed to the command
-  export Directory:    String       # The directory in which the command should be run
-  export Stdin:        String       # The file to which standard input should be connected
-  export Stdout:       LogLevel     # How should standard output be displayed during a build
-  export Stderr:       LogLevel     # How should standard error be displayed during a build
-  export Echo:         LogLevel     # Echo the command to this stream
-  export Persistence:  Persistence  # See Persistence table above
-  export LocalOnly:    Boolean      # Must run directly in the local workspace; no output detection performed
-  export Resources:    List String  # The resources a runner must provide to the job (licenses/etc)
-  export RunnerFilter: Runner => Boolean # Reject from consideration Runners which the Plan deems inappropriate
-  export Usage:        Usage        # User-supplied usage prediction; overruled by database statistics (if any)
-  export FnInputs:     (List String => List String) # Modify the Runner's reported inputs  (files read)
-  export FnOutputs:    (List String => List String) # Modify the Runner's reported outputs (files created)
-  export IsAtty:       Boolean      # Should job run in psuedoterminal
+  # The label used when showing the command during execution
+  export Label:        String
```

Comment:
Should we elaborate here on how the label is printed out with the job ID in parentheses and where this shows up? I know how the label shows up in `wake --last` and the `.in.json` and `.out.json` files and probably other places, but that isn't necessarily clear from the documentation. I don't know (off the top of my head at the moment) if we have it documented elsewhere.

Reply:
It also appears in the terminal progress bar

Reply:
Also, I think one of the best features now is that you can find the output of a job by searching for its label now.

Reply:
I took a stab at this [here](https://github.com/sifive/wake/pull/1206/commits/ed11306e48d66f3b8554cd0d79b4aa09bdee9a51). Not married to this documentation implementation, so suggestions/corrections are welcomed.